CONTRIBUTING.md

```diff
@@ -83,9 +83,9 @@ Before submitting a pull request affecting the wasi-rust subsystem:
 - if you've modified internal APIs see above re wasi-js subsystem APIs
 - clear up any issues from `cargo clippy -- -D warnings`
 - format using `cargo fmt -- --check`
-- run `yarn test` and make sure all tests pass
```

Comment:
Although this line looks wrong it should remain `yarn test` for now I think. Later, if we start adding tests in Rust, we can work out how to run them in the existing test suit is `yarn test` or separately as you have here.